### PR TITLE
fix #189: modify the log config file path

### LINKED ISSUE #189: log config file can not be read
**Is this a BUG REPORT or FEATURE REQUEST?**:

> Uncomment only one, leave it on its own line: 
>
 /kind bug
> /kind feature

**What happened**:
log config file can not be read, so log configuration can not be changed by modifying the log config file.

**What you expected to happen**:
log configuration could be changed by modifying the log config file.

**How to reproduce it (as minimally and precisely as possible)**:
Change the configuration in log config file and check the log that whether the modified configuration worked.

**Anything else we need to know?**:

**Environment**:
- Gelato(release/branch) version: latesd
- OS (e.g. from /etc/os-release): Ubuntu 16.04
- Kernel (e.g. `uname -a`): Linux ubuntu 4.4.0-62-generic
- Install tools:
- Others:

### PR DESCRIPTION
**Pull request by ThisIsClark**
**What this PR does / why we need it**:
fix #189: modify the log config file path to make it can be read in docker

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #189

**Special notes for your reviewer**:

**Release note**:

```
root@ubuntu:~/gopath/src/github.com/opensds/multi-cloud (development) # git diff
diff --git a/log.conf b/log.conf
index 49e70f4..257da45 100644
--- a/log.conf
+++ b/log.conf
@@ -1,3 +1,3 @@
 path = /var/log/multi-cloud
-level = debug
-format = [%time%] [%level%] [%filename%] [%funcName%():%lineNo%] [PID:%process%] %message%
+level = info
+format = [%time%]-[%level%]-[%filename%] [%funcName%():%lineNo%] [PID:%process%] %message%

root@ubuntu:~/gopath/src/github.com/opensds/multi-cloud (development) # cat /var/log/multi-cloud/api.INFO
[2019-10-10T10:37:57+08:00]-[INFO]-[/usr/local/go/src/runtime/proc.go] [runtime.main():200] [PID:1] noauth
[2019-10-10T10:37:57+08:00]-[INFO]-[/usr/local/go/src/runtime/proc.go] [runtime.main():200] [PID:1] filter is noauth
```

### CODE REVIEW

File: api/pkg/utils/obs/log.go [hunk 1]
```diff
@@ -35,10 +34,9 @@ const (
 	defaultLogLevel        = "info"
 	unknownHost            = "unknownhost"
 	unknownUser            = "unknownuser"
-	configFileName         = "log.conf"
+	configFileName         = "/etc/log.conf"
```
Comment (wisererik) [inline api/pkg/utils/obs/log.go]: change it to /etc/multi-cloud/multi-cloud.conf

File: docker-compose.yml [hunk 2]
```diff
@@ -96,5 +101,6 @@ services:
     image: mongo
     volumes:
       - /var/log/multi-cloud:/var/log/multi-cloud
+      - ./log.conf:/etc/log.conf
     ports:
```
Comment (wisererik) [inline docker-compose.yml]: why need conf in mongo?

File: log.conf [hunk 3]
```diff
@@ -1,3 +1,3 @@
 path = /var/log/multi-cloud
-level = debug
```
Comment (wisererik) [inline log.conf]: it's better put it into multi-cloud/example/multi-cloud.conf

File: docker-compose.yml (hunk 2 above)
Comment (ThisIsClark) [reply docker-compose.yml]: Mongo didn't need it.
I have deleted the volume mapping in mongo

File: api/pkg/utils/obs/log.go (hunk 1 above)
Comment (ThisIsClark) [reply api/pkg/utils/obs/log.go]: Done

File: log.conf (hunk 3 above)
Comment (ThisIsClark) [reply log.conf]: It seems there is no multi-cloud/example/multi-cloud.conf.

File: docker-compose.yml [hunk 4]
```diff
@@ -27,6 +27,7 @@ services:
       - /etc/ssl/certs:/etc/ssl/certs
       - /etc/localtime:/etc/localtime
       - /var/log/multi-cloud:/var/log/multi-cloud
+      - ./multi-cloud.conf:/etc/multi-cloud/multi-cloud.conf
```
[the remark this answers is not in the document]
Comment (ThisIsClark) [reply docker-compose.yml]: Done

File: api/pkg/utils/obs/log.go [hunk 5]
```diff
@@ -1,21 +1,18 @@
 package obs
```
Comment (sfzeng) [inline api/pkg/utils/obs/log.go]: Add license announcement here, please.

Comment (ThisIsClark) [reply api/pkg/utils/obs/log.go]: Done

**Pull request merged**